### ISSUE TITLE
gnt-instance moved should be disallowed with externally mirrored storage

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#439]
Originally reported of Google Code with ID 383.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".

# gnt-cluster --version
gnt-cluster (ganeti v2.6.2) 2.6.2

# gnt-cluster version
Software version: 2.6.2
Internode protocol: 2060000
Configuration format: 2060000
OS api version: 20
Export interface: 0

<b>What distribution are you using?</b>

Debian squeeze

<b>What steps will reproduce the problem?</b>
1. create instance instance1 with "-t sharedfile" on node1
2. shutdown instance1
3. run "gnt-instance move -n node2 instance1"

What is the expected output?

instance1 gets moved to node2

What do you see instead?

# gnt-instance move -n clrz04-02 vhrz496
Instance instance1 will be moved. This requires a shutdown of the
instance. Continue?
y/[n]/?: y
Wed Feb 20 13:13:24 2013  - INFO: Not checking memory on the secondary node as instance will not be started
Wed Feb 20 13:13:24 2013  - INFO: Shutting down instance instance1 on source node node1
Wed Feb 20 13:13:24 2013  - WARNING: Device creation failed, reverting...
Failure: command execution error:
Can't create block device <Disk(type=file, logical_id=('loop', '/srv/ganeti/shared-file-storage/instance1/disk0'), physical_id=('loop', '/srv/ganeti/shared-file-storage/instance1/disk0'), children=None, visible as /dev/disk/0, size=3072m)> on node node2 for instance instance1: Can't create block device: File already existing: /srv/ganeti/shared-file-storage/instance1/disk0

<b>Please provide any additional information below.</b>

Even worse:

/srv/ganeti/shared-file-storage/instance1/disk0 is *deleted* !
```
Originally added on 2013-02-20 12:25:02 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi,

Thanks for the report. gnt-instance move is in this case the wrong command: you want "migrate" or "failover". That said the behavior is wrong, and should be fixed, by disallowing gnt-instance move altogether, in this case.

Thanks,

Guido
```
Originally added on 2013-02-20 13:41:15 +0000 UTC.

Added Labels: Type-Defect
Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-02-20 14:30:33 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-02-20 14:30:46 +0000 UTC.

Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
fixed in commit 0ea861546a4badaf9e1 on devel-2.7
```
Originally added on 2013-04-04 08:13:47 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
